sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java

```diff
@@ -0,0 +1,106 @@
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestProxyTestBase;
+import com.azure.core.test.annotation.LiveOnly;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.AzurePowerShellCredentialBuilder;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.azure.resourcemanager.servicefabricmanagedclusters.models.ManagedCluster;
+import com.azure.resourcemanager.servicefabricmanagedclusters.models.Sku;
+import com.azure.resourcemanager.servicefabricmanagedclusters.models.SkuName;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class ServiceFabricManagedClustersManagerTests extends TestProxyTestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private ServiceFabricManagedClustersManager serviceFabricManagedClustersManager = null;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new AzurePowerShellCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        serviceFabricManagedClustersManager = ServiceFabricManagedClustersManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @LiveOnly
+    public void testCreateManagedCluster() {
+        ManagedCluster managedCluster = null;
+        try {
+            String clusterName = "cluster" + randomPadding();
+            String adminUser = "user" + randomPadding();
+            String adminPassWord = "!QA2ws#ED";
```

Comment:
Please check how we use the password in other live tests. Do we append random padding as well?

The reason is for security. We do not want to have a test resource with predictable password, even if this resource only exists for a few minutes.
@XiaofeiCao thoughts?

Reply:
Yes, this is valid concern and CI is reporting warning for fixed password. 
Previously we did random padding for password: https://github.com/Azure/azure-sdk-for-java/blob/b0f6934ec0a04faf5f6a7ee5767a1bdd7d5d34df/sdk/postgresqlflexibleserver/azure-resourcemanager-postgresqlflexibleserver/src/test/java/com/azure/resourcemanager/postgresqlflexibleserver/PostgreSqlManagerTests.java#L94-L95

I think this is a common scenario for our test pipelines. Let me see if eng sys channel knows anything(e.g. if they could generate a password for us)..

Reply:
@v-hongli1 Please also help fix previous password using the random padding approach. At least we could fix CredScan...
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4217712&view=logs&j=bc38e8b8-e027-53cb-48e7-2adbd1070eca&t=d1aba2c3-5b21-511d-f336-affeabd629f9&l=361

Reply:
Fixed in the new version.